src/matrix_t/matrix_t.h

```diff
@@ -29,16 +29,9 @@ class matrix_t {
     size = (cols * get_data_type_size(data_type) * channel) *
            rows; //(cols * rows);
     dt = new data_t((cols * get_data_type_size(type) * channel) * rows);
-    std::cout << (data_buffer.typeOf().as<std::string>()) << std::endl;
     if (isType(data_buffer, "object")) {
```

Comment:
That is not optimal because if you provide a `null` entry the result by typeof will be an object, but will raise an error by the fillData method because the argument is null and not a Uint8Array or whatever.

Reply:
I will try to fix this in a near future. But for now if the data_buffer is empty leave a `0` not a `null`.